packages/apollos-data-prayer/src/prayer-requests/data-source.js

```diff
@@ -94,26 +94,28 @@ export default class PrayerRequest extends RockApolloDataSource {
     }
   };
 
-  // QUERY PrayerRequests by Campus
-  getAllByCampus = async (campusId) => {
-    try {
-      const {
-        dataSources: { Auth },
-      } = this.context;
+  getAllByCampus = async (id = '') => {
+    const {
+      dataSources: { Auth, Campus },
+    } = this.context;
 
-      const { primaryAliasId } = await Auth.getCurrentPerson();
+    const { primaryAliasId } = await Auth.getCurrentPerson();
 
-      const prayers = await this.request('PrayerRequests/Public')
-        .filter(
-          `(CampusId eq ${
-            parseGlobalId(campusId).id
-          }) and (RequestedByPersonAliasId ne ${primaryAliasId})`
-        )
-        .get();
-      return this.sortPrayers(prayers);
-    } catch (err) {
-      throw new Error(err);
+    let campus;
+    let campusID;
+    if (id === '') {
+      campus = await Campus.getForPerson(primaryAliasId);
+      campusID = campus.id;
+    } else {
+      campusID = parseGlobalId(id).id;
     }
+
+    const prayers = await this.request('PrayerRequests/Public')
+      .filter(
+        `(CampusId eq ${campusID}) and (RequestedByPersonAliasId ne ${primaryAliasId})`
```

Comment:
This isn't something to fix here but this `primaryAliasId` is (I think) one of those places where we'll need to make a change to look through all of a person's `personAliasId`'s instead.